src/call/mod.rs

```diff
@@ -156,10 +156,15 @@ impl<Req, Resp> Method<Req, Resp> {
 #[derive(Debug, Clone)]
 pub struct RpcStatus {
     /// gRPC status code. `Ok` indicates success, all other values indicate an error.
-    pub status: RpcStatusCode,
+    code: RpcStatusCode,
```

Comment:
Why change the field name?

Reply:
Because I want to name the new field as `details` just like other implementations. It's going to be a breaking change, so renaming all fields to keep consistent with protobuf struct for better understanding.